### PR TITLE
Make platform mainnet settings the default

### LINKED ISSUE #9340: Make mainnet settings the default in code
The settings for mainnet (in `settings.txt` used by the platform) should be the default in the code. No overrides should be necessary for deploying to mainnet. This ticket is to update any configuration defaults so that they reflect the mainnet setting and make the `hedera-node/configuration/mainnet/settings.txt` file empty. For any defaults that change as part of this process, the correct value for all other networks should be put into the appropriate `settings.txt` file.

### PR DESCRIPTION
**Pull request by poulok**
**Description**:
Makes the mainnet settings the defaults in the code and remove values from `settings.txt`, with the exception of the `state.mainClassNameOverride`.

**Related issue(s)**:

Fixes #9340

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

Should be backported to the release 44 branch once merged.

### CODE REVIEW

File: platform-sdk/sdk/settings.txt [hunk 1]
```diff
@@ -10,10 +10,13 @@
...
                                           #    Gossip / Networking    #
                                           #############################
-
-chatter.useChatter,                       false
 sync.syncProtocolPermitCount,             2
 
+                                          #############################
+                                          #           State           #
+                                          #############################
+state.saveStatePeriod,                    60
```
Comment (Nana-EC) [inline platform-sdk/sdk/settings.txt:18]: Q: should this be 900?
I'm looking at https://github.com/hashgraph/hedera-services/pull/9341/files#diff-64ffea1d3c829bdd759986b8faf13e821cc95918a7ce7bc6f8286d87124f55ecL15

**Review by Nana-EC — COMMENTED** [review]
3 clarifications to check

**Comment by jsync-swirlds** [comment]
It appears we have at least a few unit tests that check the exact value of config entries...
`https://scans.gradle.com/s/6zkxlm5i7smvs/tests/task/:platform-sdk:swirlds-common:test/details/com.swirlds.common.metrics.config.MetricsConfigTest/testDefaultMetricsConfig()?top-execution=1`
`https://scans.gradle.com/s/6zkxlm5i7smvs/tests/task/:platform-sdk:swirlds-common:test/details/com.swirlds.common.config.export.ConfigExportTest/testPrint()?top-execution=1`

**Review by lpetrovic05 — DISMISSED** [review]

File: platform-sdk/sdk/settings.txt (hunk 1 above)
Comment (poulok) [reply platform-sdk/sdk/settings.txt:18]: This file is used when running locally. Usually when we run locally it is much better to save a state every minute than 15 minutes.

File: platform-sdk/sdk/settings.txt [hunk 2]
```diff
@@ -27,6 +30,7 @@ platformStatus.observingStatusDelay,      1ms
                                           #############################
 
 event.enableEventStreaming,               false
+event.eventsLogDir,                       ./eventsStreams
```
[the remark this answers is not in the document]
Comment (poulok) [reply platform-sdk/sdk/settings.txt:33]: This file is used when running locally, so values are adjusted from the mainnet settings.

File: platform-sdk/sdk/settings.txt [hunk 3]
```diff
@@ -35,12 +39,7 @@ event.preconsensus.enableStorage,         true
                                           #############################
 
 metrics.csvFileName,                      PlatformTesting
-
-                                          #############################
-                                          #           State           #
-                                          #############################
-
-state.saveStatePeriod,                    60
+prometheus.endpointEnabled,               false
```
[the remark this answers is not in the document]
Comment (poulok) [reply platform-sdk/sdk/settings.txt:42]: This file is used when running locally, so values are adjusted from the mainnet settings.

**Review by jsync-swirlds — APPROVED** [review]

**Review by Nana-EC — APPROVED** [review]

**Review by timo0 — APPROVED** [review]

**Review by SimiHunjan — APPROVED** [review]

**Review by nathanklick — APPROVED** [review]

**Pull request merged**